actionpack/lib/action_dispatch/http/request.rb

```diff
@@ -153,6 +153,14 @@ def routes=(routes) # :nodoc:
       set_header("action_dispatch.routes", routes)
     end
 
+    def route_uri_pattern # :nodoc:
```

Comment:
```suggestion
    def route_uri_pattern
```

We want this method to be public API so we need to document it.